docs/source/index.rst

```diff
@@ -14,6 +14,8 @@ You can download the latest version from `GitHub <https://github.com/JuBiotech/b
 
 In the following chapters, we introduce the data structures and analysis features.
 
+Go and take a look to accelerate your BioLector data analysis!
```

Comment:
This looks good, but please add context.

Reply:
Let's do this next time :)